.github/workflows/ci.yaml

```diff
@@ -6,7 +6,7 @@ jobs:
     name: Build and Test
     strategy:
       matrix:
-        go: ['1.11', '1.12', '1.13', '1.14', '1.15', '1.16', '1.17']
+        go: ['1.18']
```

Comment:
nit: `1.18.x` so you get the latest bugfix version

Reply:
also, I would personally test on 1.17.x as well - dropping support for it already seems premature, and I have multiple modules using quicktest which support two Go versions.

Reply:
Err, right, generics. D'oh.